runtime/lua/vim/lsp/tagfunc.lua

```diff
@@ -0,0 +1,73 @@
+local lsp = vim.lsp
+local util = vim.lsp.util
+
+local function mk_tag_item(name, range, uri, offset_encoding)
+  local bufnr = vim.uri_to_bufnr(uri)
+  local byte = util._get_line_byte_from_position(bufnr, range.start, offset_encoding)
```

Comment:
For me column positions are now off by one

Reply:
Ah need to convert back to 0-indexed

Reply:
Should be fixed